app/frontend/src/components/commons/Modal/Footer.tsx

```diff
@@ -0,0 +1,31 @@
+import * as styles from './index.css';
+import { Button } from '../Button';
+
+type FooterProps = {
+  buttonType: 'single' | 'double';
+  confirmButtonText: string;
+  cancelButtonText: string;
+  onClickConfirm: () => void;
+};
+
+export function Footer({ buttonType, confirmButtonText, cancelButtonText, onClickConfirm }: FooterProps) {
+  return (
```

Comment:
p2;

```suggestion
export function Footer({ buttonType, confirmButtonText, cancelButtonText, onClickConfirm }: FooterProps) {

  const closeModal = () => {
    onClickConfirm?.();
    // 모달 닫는 로직
  }
  return (
```

이런 식으로 전달받은 onClickConfirm을 실행하고 나서 모달을 닫는 액션을 취해야 하지 않을까요? 함수 이름은 임시로..